drivers/mtd/mtd.c

```diff
@@ -122,11 +134,49 @@ int mtd_write(mtd_dev_t *mtd, const void *src, uint32_t addr, uint32_t count)
     const uint32_t page_shift = bitarithm_msb(mtd->page_size);
     const uint32_t page_mask = mtd->page_size - 1;
 
-    return mtd_write_page(mtd, src, addr >> page_shift, addr & page_mask, count);
+    return mtd_write_page_raw(mtd, src, addr >> page_shift, addr & page_mask, count);
+}
+
+#ifdef MODULE_MTD_HL_WRITE
```

Comment:
Maybe rename the module, now that the function has been renamed?

Reply:
Good idea, changed it.